js/contextMenus.js

```diff
@@ -1093,7 +1093,8 @@ function mainTemplateInit (nodeProps, frame) {
   })
 
   const passwordManager = getActivePasswordManager()
-  if (passwordManager.get('extensionId')) {
+  if (passwordManager.get('extensionId') &&
+    passwordManager.get('displayName') === 'Dashlane') {
```

Comment:
we can just remove these then. The only reason we created context menus for the password managers was because we didn't have contextMenu api support.